### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 44

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#232] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255432-d6984b2e-40e6-491b-a5db-7fc8a1df7519.png)
![image](https://user-images.githubusercontent.com/69389461/152255449-70ebc9cf-4e0d-467c-af07-76b09f145d7a.png)

**Comment by i0nut02** [comment]
1.  26^4 * 10^3 * 1
2. 26^3 * 10^3 * 3 + 26^4 * 10^3 * 1 - 26^3 * 10^3
3. 4 * 25^3 * 3 * 9^2 * 3     (ho considerato le targhe password)
